MekHQ/src/mekhq/gui/view/ForceViewPanel.java

```diff
@@ -71,6 +76,8 @@
 	private javax.swing.JLabel lblTech1;
 	private javax.swing.JLabel lblTech2;
 	
+	private static Parser parser = Parser.builder().build();
```

Comment:
I wasn't sure of the best way to do this. This the parser for the commonmark library. The example creates it dynamically, but that seemed like a waste to do every time this is called so I created it as a static variable in each View Panel. I would appreciate feedback about whether this is a smart way to do it.

Reply:
Ok following @sixlettervariables advice I have moved the rendering into its own static class which calls `getRenderedHTML`. This will make it easier in the future if we want to extend the way we do rendering.